src/pages/task/ActionButtons.tsx

```diff
@@ -130,13 +136,31 @@ export const ActionButtons = ({
     },
   });
 
+  const [
+    overrideTaskDependencies,
+    { loading: loadingOverrideTaskDependencies },
+  ] = useMutation<
+    OverrideTaskDependenciesMutation,
+    OverrideTaskDependenciesMutationVariables
+  >(OVERRIDE_TASK_DEPENDENCIES, {
+    onCompleted: () => {
+      dispatchToast.success("Successfully overrode task dependencies");
+    },
+    onError: (err) => {
+      dispatchToast.error(`Error overriding task dependencies: ${err.message}`);
+    },
+  });
+
   const disabled =
     loadingAbortTask ||
     loadingRestartTask ||
     loadingSetPriority ||
     loadingUnscheduleTask ||
-    loadingScheduleTask;
+    loadingScheduleTask ||
+    loadingOverrideTaskDependencies;
 
+  const canOverrideDependencies =
+    status === TaskStatus.Blocked || status === TaskStatus.WillRun;
```

Comment:
The condition to enable this button is slightly different in the legacy ui: https://github.com/evergreen-ci/evergreen/blob/main/service/templates/task.html#L145. Do you think we need to reutilize any part of the legacy condition here?

The other buttons' enabled state is fetched from evergreen to make it easier to leverage EVG constants https://github.com/evergreen-ci/evergreen/blob/main/graphql/schema.graphql#L757-L761.
It could be a good idea to create a canOverrideDependencies resolver in case we need compare a value against a permission constant like we do in the legacy UI.

Reply:
Great Idea! Just implemented